### ISSUE TITLE
Editable dataTable with Buttons Not Keyboard Accessible

### ISSUE THREAD

**Issue by outlinedspider** [primefaces/primefaces#1782]
# IMPORTANT
- If you open a issue, fill every item. Otherwise the issue might be closed as invalid.
- Before you open a issue, test it with the current/newest version.
## 1) Environment
- PrimeFaces version: 6.0.4
- Does it work on the newest PrimeFaces version?  No
- Application server + version: Glassfish 4.1.1
- Affected browsers:  Firefox 48.0
## 2) Expected behavior

Editable dataTable "buttons" should be keyboard focusable and not empty spans.  This means that the dynamically created buttons need to actually be buttons.  Spans are not keyboard accessible.
## 3) Actual behavior

Dynamically creates empty spans that are visually made to look like buttons.
## 4) Steps to reproduce

on primefaces showcase, go to editable dataTables, view the page source for the edit/cancel/delete/save "button" and you will see they're empty spans.
## 5) Sample XHTML

<td style="width:32px" role="gridcell">

<div id="form:cars1:0:j_idt108" class="ui-row-editor ui-helper-clearfix">
<span class="ui-icon ui-icon-pencil"></span>
<span class="ui-icon ui-icon-check"></span>
<span class="ui-icon ui-icon-close"></span>
</div>

</td>
## 6) Sample bean

taken from primefaces showcase... the sample bean is in your system

**Comment by outlinedspider** [comment]
Fixed in 6.0.4
